nixos/modules/rename.nix

```diff
@@ -257,4 +257,9 @@ with lib;
          The prometheus exporters are now configured using `services.prometheus.exporters'.
          See the 18.03 release notes for more information.
        '' ));
+
+    # Fontconfig
+    (mkRemovedOptionModule [ "fonts" "fontconfig" "dpi" ] "Use display server-specific options")
+  ];
```

Comment:
GitHub syntax highlighting shows that this does not parse.

Reply:
Good catch, bad rebase. Fixed!